### PR TITLE
[CSM-525] More public logs in wallet

### PR DESCRIPTION
**Pull request by Martoon-00**
Log errors and other interesting stuff in wallet.
This PR contains bunch of small improvements in logging, please see commit history for details

**This PR enables very important logging, diagnosis of wallet-related issues is hindered without it**
**This PR blocks #1874**

### CODE REVIEW

File: node/src/Pos/Wallet/Web/Pending/Types.hs [hunk 1]
```diff
@@ -89,6 +90,9 @@ instance Buildable PtxCondition where
         PtxWontApply reason _ ->
             bprint ("wont apply ("%build%")") reason
 
+instance Buildable (SecureLog PtxCondition) where
+    build = buildUnsecure
```
Comment (gromakovsky) [inline node/src/Pos/Wallet/Web/Pending/Types.hs]: I have no idea what's going on, but it looks suspicious that `Buildable` instance for `SecureLog whatever` uses `unsecure`.

File: wallet/src/Pos/Wallet/Web/Methods/History.hs [hunk 2]
```diff
@@ -39,24 +39,15 @@ import           Pos.Wallet.Web.State       (AddressLookupMode (Ever), addOnlyNe
...
 getFullWalletHistory :: MonadWalletWebMode m => CId Wal -> m (Map TxId (CTx, POSIXTime), Word)
 getFullWalletHistory cWalId = do
     addrs <- mapM decodeCTypeOrFail =<< getWalletAddrs Ever cWalId
-
     unfilteredLocalHistory <- getLocalHistory addrs
-
-    blockHistory <- getHistoryCache cWalId >>= \case
-        Just hist -> pure hist
-        Nothing -> do
-            logWarning $
```
Comment (gromakovsky) [inline wallet/src/Pos/Wallet/Web/Methods/History.hs]: Should we worry if it's empty? 🤔

Comment (Martoon-00) [reply wallet/src/Pos/Wallet/Web/Methods/History.hs]: No, it's common case for newly created wallets. Don't understand your concern

File: node/src/Pos/Wallet/Web/Pending/Types.hs (hunk 1 above)
Comment (Martoon-00) [reply node/src/Pos/Wallet/Web/Pending/Types.hs]: Fixed

File: wallet/src/Pos/Wallet/Web/Methods/History.hs (hunk 2 above)
Comment (gromakovsky) [reply wallet/src/Pos/Wallet/Web/Methods/History.hs]: I mean: was this `logWarning` here by mistake or you just deleted it for another reason? So apparently it was here by mistake, ok.

Comment (Martoon-00) [reply wallet/src/Pos/Wallet/Web/Methods/History.hs]: Ah, yep

**Comment by Martoon-00** [comment]
Waiting for https://iohk.myjetbrains.com/youtrack/issue/DEVOPS-565 to be resolved..

File: node/src/Pos/Wallet/Web/State/State.hs [hunk 3]
```diff
@@ -181,7 +181,7 @@ getUpdates = queryDisk A.GetUpdates
 getNextUpdate :: WebWalletModeDB ctx m => m (Maybe CUpdateInfo)
 getNextUpdate = queryDisk A.GetNextUpdate
 
-getHistoryCache :: WebWalletModeDB ctx m => CId Wal -> m (Maybe (Map TxId TxHistoryEntry))
+getHistoryCache :: WebWalletModeDB ctx m => CId Wal -> m (Map TxId TxHistoryEntry)
```
Comment (adinapoli-iohk) [inline node/src/Pos/Wallet/Web/State/State.hs]: Oh, this looks outside the changes of simply logging. What was this necessary?

Comment (Martoon-00) [reply node/src/Pos/Wallet/Web/State/State.hs]: Strangely but it somehow relates.
Previously as part of one fix all history-related functions to access / modify database were rewritten in a way that, if wallet doesn't exist, empty history for it is returned.
But this change was partly lost during rebase, and now we have very annoying but inappropriate warning here: https://github.com/input-output-hk/cardano-sl/pull/1858/files#diff-1bfb54b702d6d93310adefeed510d053L52
(@fersel complained on it twice already)

I can create separate PR if needed where this change will be adopted

**Comment by Martoon-00** [comment]
Logs duplication is removed.
Also separate PR was created for db fix: https://github.com/input-output-hk/cardano-sl/pull/2188, once we merge it only logging-related changes will remain here.

Please note that this PR **must** be merged before we give release branch to bittrex, because all wallet logs has been directed to secret logs since `cardano-sl-1.0-exchanges` (except servant logs), and this PR brings them at least partly back.

**Pull request closed without merging**